numpy/core/fromnumeric.py

```diff
@@ -63,10 +82,46 @@ def _wrapfunc(obj, method, *args, **kwds):
     # in its class, but its signature is not identical to that
     # of NumPy's. This situation has occurred in the case of
     # a downstream library like 'pandas'.
-    except (AttributeError, TypeError):
+    except (AttributeError, TypeError) as e:
```

Comment:
That doesn't seem to be the case for at least some of the functions where you are calling `_wrapfunc2`. What particular functions are the problem?